atomic_reactor/plugins/add_help.py

```diff
@@ -89,13 +90,19 @@ def render_help_file(self, build_dir: BuildDir) -> List[Path]:
 
         help_path = build_dir.path / self.help_file
 
+        osbs = get_openshift_session(self.workflow.conf,
+                                     self.workflow.namespace)
+        build_info = osbs.get_build(self.workflow.pipeline_run_name)
+        start_time = build_info['status']['startTime']
+        start_ts = dt.strptime(start_time, '%Y-%m-%dT%H:%M:%SZ').timestamp()
+
         with open(help_path, 'r+') as help_file:
             lines = help_file.readlines()
 
             if not lines[0].startswith("% "):
                 lines.insert(0, "%% %s (1) Container Image Pages\n" % name)
                 lines.insert(1, "%% %s\n" % maintainer)
-                lines.insert(2, "%% %s\n" % dt.fromtimestamp(atomic_reactor_start_time)
+                lines.insert(2, "%% %s\n" % dt.fromtimestamp(start_ts)
```

Comment:
this part converts from datetime to timestamp and then back again

Reply:
removed reconversion